docs/model_cookbook.md

```diff
@@ -257,6 +257,16 @@ iree-run-module \
   --parameters=model=/tmp/open_llama_3b_v2/open-llama-3b-v2-f16.gguf
 ```
 
+## Evaluation pipeline
+
+Run perplexity test:
+
+```bash
+python -m  sharktank.evaluate.perplexity \
+  --gguf-file=llama8b_f16.gguf \
+  --tokenizer-config-json=tokenizer_config.json
+```
```

Comment:
Want to include pytest instructions here too / instead? That will help if more tests get added (e.g. other models / variants)